Measurement[] camera_measurements
Measurement[] joint_measurements
Measurement[] laser_measurements
Command[] joint_commands
string sample_id
string target_id
string chain_id
---
calibration_msgs/RobotMeasurement result
---
calibration_msgs/IntervalStatus status
string progress


